### DISCUSSION TITLE
how to login default admin when using oauth??

### DISCUSSION THREAD

**Discussion by JuHyung-Son** [apache/airflow#28211] [Q&A] [answered]
I`m trying to apply google oauth2
but then, I cannot login admin, so i can not set my accounts as admin.

<img width="1558" alt="image" src="https://user-images.githubusercontent.com/20140126/206338410-1edfba32-7055-4be6-8883-4011b48ee30e.png">

**Comment by potiuk** [comment, marked as answer]
Use CLI
